tools/shared.py

```diff
@@ -2347,6 +2344,9 @@ def acorn_optimizer(filename, passes, extra_info=None, return_output=False):
   # evals ctors. if binaryen_bin is provided, it is the dir of the binaryen tool for this, and we are in wasm mode
   @staticmethod
   def eval_ctors(js_file, binary_file, binaryen_bin='', debug_info=False):
+    if Settings.WASM_BACKEND:
+      logger.debug('Ctor evalling in the wasm backend is disabled due to https://github.com/emscripten-core/emscripten/issues/9527')
```

Comment:
Should we at least warn so the user knows they have a useless flag?

Reply:
I don't want to explicitly write "remove -s EVAL_CTORS" because we do hope to fix it, and if they remove it they might not remember to restore it. So just a debug mode message seems safer to me.